src/routes/common/transformers.js

```diff
@@ -20,7 +20,13 @@ const transformGifResponse = (gif) => ({
 	
 })
 
+const transformGifLikeResponse = (gifLike) => ({
+	userId: gifLike.userId
+	
+})
+
 module.exports = {
 	transformArticleResponse,
-	transformGifResponse
+	transformGifResponse,
+	transformGifLikeResponse
```

Comment:
@beblicarl we use the `common` to signify we're using the transformer in different places. This will not need a function. you could just transform it directly there in the response

Reply:
yes please